bbn-test-4/finality-providers/scripts/verify-new-fp-offchain.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -eu
+
+# USAGE:
+# ./verify-new-fp-offchain.sh
+
+# checks for modified files in **/bbn-test-4/finality-providers/** compared to main branch
+# and validates if the finality provider registration has valid values to send the
+# transaction to BTC chain.
+
+CWD="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. $CWD/fp-changed.sh
+
+EOTSD_BIN="${EOTSD_BIN:-eotsd}"
+STAKERCLI_BIN="${STAKERCLI_BIN:-stakercli}"
+
+if ! command -v jq &> /dev/null; then
+  echo "⚠️ jq command could not be found!"
+  echo "Install it by checking https://stedolan.github.io/jq/download/"
+  exit 1
+fi
+
+if ! command -v $EOTSD_BIN &> /dev/null; then
+  echo "⚠️ $EOTSD_BIN command could not be found!"
+  echo "Install it by checking https://github.com/babylonchain/finality-provider.git"
```

Comment:
Let's give tags for the above.

Reply:
https://github.com/babylonchain/networks/pull/193/commits/0005a0d9dc0407f9edfce31c2eddd42f861a562d